docs/articles/documentation/templates/authentication/role-options.md

```diff
@@ -0,0 +1,33 @@
+### options.preserveUrl
+
+Use this option to control which page is opened after you switch to the role.
+
+By default, TestCafe navigates back to the page that was opened previously to switching to the role.
+Set the `preserveUrl` option to `true` to save the URL to which the browser was redirected after logging in.
+TestCafe navigates to the saved URL each time you switch to the role.
+
+Use this option to prevent losing session-related data (like session ID) stored in the URL.
```

Comment:
Change to:
Use this option to prevent the loss of session-related data stored in the URL (e.g., session ID).